tests/gagarin/tuxlab.api.js

```diff
@@ -1,3 +1,94 @@
 /** Test Suite for TuxLab-App
     using Meteor Gagarian (https://github.com/anticoders/gagarin)
 **/
+describe('tuxlab.task tests',function(){
+  var tuxlab = require('../../server/api/tuxlab.js');
```

Comment:
Is there a reason you need to import this file separately in each test?  Can you just import it once at the top?